File: baekjoon/p11008/Main.java
package baekjoon.p11008;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();

        while (t-- > 0) {
            System.out.println(solve(sc.next(), sc.next()));
        }
    }

    private static int solve(String s, String p) {
        return s.replaceAll(p, " ").length();
    }

}

File: baekjoon/p11008/Main.kt
package baekjoon.p11008

fun main() {
    val t = readLine()!!.toInt()
    repeat(t) {
        val inputs = readLine()!!.split(" ")
        println(solve(inputs[0], inputs[1]))
    }
}

private fun solve(s: String, p: String): Int {
    return s.replace(p, " ").length
}